bounties_api/std_bounties/seo_client.py

```diff
@@ -0,0 +1,56 @@
+from bounties import settings
+from bounties.sns_client import sns_publish
+from bounties.utils import base_url_for, bounty_url_for, profile_url_for
+from std_bounties.models import Bounty
+from user.models import User
+from uuid import uuid4
+
+
+class SEOClient:
```

Comment:
There is no reference to `self` in this file - would it make more sense as functions? We are creating these clients each time we just need to call 1 function, which has no concept of state. Even though it's not that important, efficiency wise (it's all fast enough), it says to a reader that there is initialization and state involved.